cicd/missed_docstrings.txt

```diff
@@ -1,13 +1,13 @@
 credsweeper/common/keyword_checklist.py:7:0: C0115: Missing class docstring (missing-class-docstring)
 credsweeper/config/config_manager.py:11:0: C0115: Missing class docstring (missing-class-docstring)
 credsweeper/config/config_manager.py:14:4: C0116: Missing function or method docstring (missing-function-docstring)
-credsweeper/credentials/candidate.py:12:0: C0115: Missing class docstring (missing-class-docstring)
-credsweeper/credentials/candidate.py:34:4: C0116: Missing function or method docstring (missing-function-docstring)
-credsweeper/credentials/candidate.py:42:4: C0116: Missing function or method docstring (missing-function-docstring)
-credsweeper/credentials/candidate.py:50:4: C0116: Missing function or method docstring (missing-function-docstring)
-credsweeper/credentials/candidate.py:58:4: C0116: Missing function or method docstring (missing-function-docstring)
-credsweeper/credentials/candidate.py:66:4: C0116: Missing function or method docstring (missing-function-docstring)
-credsweeper/credentials/candidate.py:74:4: C0116: Missing function or method docstring (missing-function-docstring)
+credsweeper/credentials/candidate.py:13:0: C0115: Missing class docstring (missing-class-docstring)
+credsweeper/credentials/candidate.py:35:4: C0116: Missing function or method docstring (missing-function-docstring)
+credsweeper/credentials/candidate.py:43:4: C0116: Missing function or method docstring (missing-function-docstring)
+credsweeper/credentials/candidate.py:51:4: C0116: Missing function or method docstring (missing-function-docstring)
+credsweeper/credentials/candidate.py:59:4: C0116: Missing function or method docstring (missing-function-docstring)
```

Comment:
Setters-getters might be commented simple. It helps us reduce documentation loan.

Reply:
I added them by [13bfe52](https://github.com/Samsung/CredSweeper/pull/159/commits/13bfe522db92016b084ef8d261e88860f5ee34fe).
I'll make other PR for regarding docstrings.

I should to clean them all. :broom: